deployments/puppet/metadata.json

```diff
@@ -54,12 +62,19 @@
         "14.04",
         "16.04"
       ]
+    },
+    {
+      "operatingsystem": "Windows",
+      "operatingsystemrelease": [
+        "2012 R2",
+        "2016"
+      ]
     }
   ],
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.7.0 < 6.0.0"
+      "version_requirement": ">= 4.7.0 <= 6.6.0"
```

Comment:
Will it not work with puppet newer than 6.6.0?

Reply:
Our tests use 6.6.0. @jchengsfx thoughts?

Reply:
I've tested manually with 6.2.0, and integration tests for 5.0.0 (oldest from chocolatey) and 6.6.0 (current latest from chocolatey) have been successful.

Reply:
Updated to ">= 5.0.0"